src/query/combine.rs

```diff
@@ -167,7 +167,7 @@ mod tests {
                 .left_join(fruit::Entity)
                 .select_also(fruit::Entity)
                 .filter(cake::Column::Id.eq(1))
-                .filter(fruit::Column::Id.eq(2))
+                .filter(ColumnTrait::eq(&fruit::Column::Id, 2))
```

Comment:
I had to specify ColumnTrait here as I've added `#[derive(PartialEq)]` to `fruit::Column` to test the `FromStr` results.

Reply:
Thank you. I understand now. It makes the 'eq' method ambiguous I suppose.